src/betterproto/__init__.py

```diff
@@ -1027,6 +1065,80 @@ def parse(self: T, data: bytes) -> T:
 
         return self
 
+    def from_stream(self: T, stream: BinaryIO, size: int = None) -> T:
+        """
+        Parse the binary encoded Protobuf from the provided stream into this message
+        instance. This returns the instance itself and is therefore assignable and
```

Comment:
Probably better to use message instead of Protobuf here:
```suggestion
        Parse the binary encoded message from the provided stream.
        This returns the instance itself and is therefore assignable and
```